components/EmbedPage/EmbedPage.tsx

```
@@ -1,9 +1,79 @@
+      </div>
+    );
   return (
-    <div className="size-full flex items-center justify-center">
-      <p className="bg-grey-moss-100 px-3 py-2 font-archivo text-3xl text-center">
-        Coming soon
-      </p>
+    <div className="size-full p-2 flex flex-col items-center">
+      <div className="relative flex flex-col w-full grow overflow-hidden">
+        <div className="size-full h-fit absolute z-[2] left-0 top-0 border border-white">
+          <div
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)